README.md

```diff
@@ -41,121 +46,187 @@ You can find a version of this docummentation, properly rendered, at https://doc
+You can also access it directly at [http://localhost:8080](http://localhost:8080).
 
-If you prefer to stay with the command line, you can execute:
+##### Query command-line interface (CLI)
+
+If you prefer to work within your terminal via command line, you can open a SQL REPL
+that allows you to execute queries against your repositories by executing:
 
 ```bash
+# Launch the query CLI REPL
 srcd sql
 ```
 
-This will open a SQL REPL that allows you to execute queries against your repositories.
-
 If you want to run a query directly, you can also execute it as such:
 
 ```bash
+# Run query via CLI
 srcd sql "SHOW tables;"
 ```
 
-You might have noticed that some queries below use the UAST function. This is to transform code to a [Universal Abstract Syntax Tree](#babelfish-uast). If you want a playground to see examples of the UAST, or run your own, you can launch the parse web client.
+**Note:**
+Queries using the [UAST](#babelfish-uast) function are meant for the `web sql` interface.
+The column for UAST will be seen as binary data in the CLI.
+
+#### Parsing code
+
+Sometimes you may want to parse files directly as [UASTs](#babelfish-uast).
 
-**Note:** queries using the UAST function are meant for the `web sql` interface. The column for UAST will be seen as binary data in the CLI.
+To see which languages are available, check the table of [supported languages](#babelfish-uast).
 
-To see which languages are available, check the table of [supported languages](https://docs.sourced.tech/babelfish/languages).
+##### Parse web client
 
-The first time you launch the web client, it will download and install the recommended drivers (those that are beta or higher). 
+If you want a playground to see examples of the UAST, or run your own, you can launch the [parse web client](https://github.com/bblfsh/web).
+
+The first time you launch the web client, it will download and install the recommended language drivers (those that are beta or higher):
 
 ```bash
-# Launch the web client
+# Launch the parse web client
 srcd web parse
 ```
 
-Alternatively you can also start parsing files on the command line:
+This should open the [web interface](https://github.com/bblfsh/web) in your browser.
+You can also access it directly at [http://localhost:8081](http://localhost:8081).
+
+##### Parse command-line interface (CLI)
+
+Alternatively, you can also start parsing files on the command line:
 
 ```bash
+# Parse file via CLI
 srcd parse uast /path/to/file.java
 ```
 
-#### 5. Start executing queries
+To parse a file specifying the programming language:
+
+```bash
+srcd parse uast --lang=LANGUAGE /path/to/file
+```
+
+To see the installed language drivers:
+
+```bash
+srcd parse drivers list
+```
+
+To install other language drivers use:
+
+```bash
+srcd parse drivers install python
+```
+
+### 5. Start executing queries
+
+**Understand which tables are available to you to query:**
```

Comment:
"are available to you to query" sounds a bit off. Maybe this could work better?
```suggestion
**Understand which tables are available for you to query:**
```